common/utils.py

```diff
@@ -55,3 +54,37 @@ def reddit_post_is_relevant(
         return True
 
     return False
+
+
+def write_raw_submissions_to_csv(target_filename: str, raw_submissions: List[RawSubmission]) -> None:
```

Comment:
just cut and pasted from another file

Reply:
We should just turn this into a common function instead then, I think?

Reply:
Oh you did. I'm an idiot